client.py

```diff
@@ -84,30 +84,30 @@ def parse_ServerResponse(self, response):
 if args.verbose:
     print(res1)
 
-session_id = res1.getroot().attrib['SessionID']
```

Comment:
I think this is also needed -- ET.fromstring, at least in py36, returns an Element, not an ElementTree

Reply:
Thanks. Appears to be the same back to at least [v2.7](https://docs.python.org/2.7/library/xml.etree.elementtree.html) and [v3.3](https://docs.python.org/3.3/library/xml.etree.elementtree.html)… not sure how how this worked for me :frowning_face:

Reply:
Right, the following diff line is the replace.  I clicked the wrong line / github has poor code review UI :-P.